docs/faults.rst

```diff
@@ -31,3 +31,12 @@ View BIG-IP Faults
     - On click, each row of the datagrid will display a detailed view for the fault.
     - To see older errors/warnings, SSH to the APIC server and check the log files in /data2/logs/F5Networks_F5ACIServiceCenter/faults/<BIG-IP IP>.log
     - The F5 ACI ServiceCenter currently displays the logs in UTC timezone which is the default timezone of this app.
+
+------------------ 
+
+FAQ
+===============================================
+
+**Q. I’m able to see only the last 100 errors/warnings in 'View Faults'. How can I see older faults for my BIG-IP? (View Faults feature is available in versions 2.4+)**
```

Comment:
..."I'm only able to see the last 100 errors/warnings in 'View Faults' (View Faults is available in versions 2.4+). How can I see older faults for my BIG-IP?"
or just move the phrase in parenthesis to the answer section..